src/amp-story-player/amp-story-player-impl.js

```diff
@@ -386,10 +386,16 @@ export class AmpStoryPlayer {
   initializeAnchorElStories_() {
     const anchorEls = toArray(this.element_.querySelectorAll('a'));
     anchorEls.forEach((element) => {
+      const posterImgEl = element.querySelector(
+        'img[data-amp-story-player-poster-img]'
```

Comment:
This shouldn't have the `data-` in the attribute right (looking at #30753)?

Reply:
It should, we ended up bikeshedding on the name and changing it do `data-*` to make it comply with standard HTML spec.

You can verify by looking at the AMP version validator
https://github.com/ampproject/amphtml/blob/22da71310603f436c1665b784c51f06b7ef092f4/extensions/amp-story-player/validator-amp-story-player.protoascii#L93

Reply:
Oh ok, I had a feeling that `data` attributes should have a value (not just the empty string) but the documentation seems to be vague on this. LGTM